src/main/java/com/zufar/onlinestore/payment/service/PaymentMethodCreator.java

```diff
@@ -0,0 +1,31 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.PaymentMethod;
+import com.stripe.param.PaymentMethodCreateParams;
+import com.zufar.onlinestore.payment.config.StripeConfiguration;
+import com.zufar.onlinestore.payment.converter.PaymentMethodConverter;
+import com.zufar.onlinestore.payment.dto.CreatePaymentMethodDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentMethodCreator {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done